lib/FilesHooks.php

```diff
@@ -1331,6 +1331,12 @@ private function getUnrelatedUsers(int $fileId, array $cachedMounts): array {
 		// now that we have a list of eventuals filtered users, we confirm they have no access to the file
 		$filteredUsers = [];
 		foreach ($usersToCheck as $userId) {
+			if (!array_key_exists($userId, $cachedPath) || $cachedPath[$userId] === null) {
+				$this->logger->notice('could not find user in list of cached path', ['cachePath' => $cachedPath, 'usersToCheck' => $usersToCheck, 'current' => $userId]);
```

Comment:
I don't know whether it is an error or not. Shouldn't we try to fetch the user path?